scraping/spiders/canada_immigration_services.py

```diff
@@ -0,0 +1,99 @@
+import re
+import logging
+
+import scrapy
+import hjson
+
+
+log = logging.getLogger(__name__)
+
+
+DEFAULT_CATEGORY = 'Any Help'
```

Comment:
I think we could move `DEFAULT_CATEGORY` to `config.constants` because it's the same for all spiders

Reply:
Yeah, makes total sense.